tests/glibc/Makefile

```diff
@@ -41,7 +41,9 @@ build: glibc
 PWD=$(shell pwd)
 
 $(APPDIR):
-	$(APPBUILDER) -i mystikos/glibc-test:0.1
+	# $(APPBUILDER) -i mystikos/glibc-test:0.1
+	# todo: replace with pre-built image
+	$(APPBUILDER) -i hullcritical/glibc-test:0.1
```

Comment:
why did you change from `mystikos/glibc-test:0.1` to `hullcritical/glibc-test:0.1`?

Reply:
This image (uploaded to my personal account, will talk to @rs-- to have it replaced with a Mystikos one), is created from the modified Dockerfile to reflect the change to glibc test runner